kitsune/urls.py

```diff
@@ -84,6 +85,9 @@
 
     # These API urls include both v1 and v2 urls.
     (r'^api/', include('kitsune.users.urls_api')),
+
+    # contribute.json url
+    url(r'^(?P<path>contribute\.json)$', servestatic, kwargs={'document_root': settings.ROOT}),
```

Comment:
Thanks for the tip @safwanrahman. I like to have things that reply on `/` on the very top `urls.py` namespace, otherwise it's harder to find like in this case.  It seems that we have 404/500 handlers here too.  I'd probably move `robots.txt` here in the future.